main_logic/omni_offline_client.py

```
@@ -271,58 +292,84 @@ async def stream_text(self, text: str) -> None:
                             
                             if content and content.strip():
-                                assistant_message += content
+                                for char in content:
+                                    if char == '|':
+                                        pipe_count += 1
+                                        if pipe_count >= 2:
+                                            pipe_positions = [i for i, c in enumerate(content) if c == '|']
+                                            if len(pipe_positions) >= 2:
+                                                content = content[:pipe_positions[1]]
+                                            fence_triggered = True
+                                            logger.info("OmniOfflineClient: 围栏触发 - 检测到第二个 | 字符，截断输出")
```

Ruff (0.15.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)